build.py

```diff
@@ -169,6 +171,52 @@ def validate_actions(user_actions, valid_targets):
             sys.exit(1)
 
 
+def generate_mo(mo_config):
+    extraction_target = getDictValue(mo_config, 'extraction_target')
```

Comment:
This method accepts a dictionary, which is very generic. Can it accept actual parameters so that callers can more easily discover and use the method correctly?

Reply:
This method is created for only for testing. I would stay with dictionary type since it is easier to add or remove properties. Can we just put comment defining argument spec?

Reply:
fixed as discussed.